src/main/java/uk/nhs/tis/trainee/actions/mapper/ActionMapper.java

```diff
@@ -118,7 +135,7 @@ public interface ActionMapper {
    * @return The ActionBroadcastDto.
    */
   @Mapping(target = "id", expression = "java(action.id().toString())")
-  @Mapping(target = "type", ignore = true)
+  @Mapping(target = "type", expression = "java(action.type().toString())")
```

Comment:
Included in the action broadcast for delete events. Assumes that downstream listeners will ignore unknown json properties. Can be reverted - it just makes the logging in the EventPublishingService clearer